compressor/storage.py

```diff
@@ -33,13 +33,12 @@ def created_time(self, name):
     def modified_time(self, name):
         return datetime.fromtimestamp(os.path.getmtime(self.path(name)))
 
-    def get_available_name(self, name, max_length=None):
-        """
-        Deletes the given file if it exists.
-        """
-        if self.exists(name):
-            self.delete(name)
-        return name
+    def save(self, filename, content):
+        temp_filename = super().save(filename, content)
+        if "_" in temp_filename:
```

Comment:
Why not compare `temp_filename` with `filename` instead ?

Reply:
Ah, of course! Thanks – fixed!